src/html/components/box-one.html

```diff
@@ -0,0 +1,13 @@
+<div class="box box-one">
+  <p class="main-thought">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</p>
+
+  <p>Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<sup><a href="">1</a></sup> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<sup><a href="">2</a></sup></p>
+
+  <blockquote class="quote">
+	<p class="quote-text">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
```

Comment:
Looks like there's a tab here instead of spaces, so it produces a weird indent on Github:
![image](https://user-images.githubusercontent.com/283397/41174550-7aa60a16-6b28-11e8-8969-0762feb36c96.png)